src/overlays/actors/ovl_Bg_Spot01_Objects2/z_bg_spot01_objects2.c

```diff
@@ -111,8 +111,8 @@ void func_808AC2BC(BgSpot01Objects2* this, GlobalContext* globalCtx) {
                 if (gSaveContext.nightFlag == 0) {
                     func_808AC22C(globalCtx->setupPathList, &position, ((s32)thisx->params >> 8) & 0xFF, 0);
                     Actor_SpawnAsChild(&globalCtx->actorCtx, thisx, globalCtx, ACTOR_EN_DAIKU_KAKARIKO, position.x,
-                                        position.y, position.z, thisx->posRot.rot.x, thisx->posRot.rot.y,
-                                        thisx->posRot.rot.z, ((((s32)thisx->params >> 8) & 0xFF) << 8) + 1);
+                                       position.y, position.z, thisx->posRot.rot.x, thisx->posRot.rot.y,
+                                       thisx->posRot.rot.z, ((((s32)thisx->params >> 8) & 0xFF) << 8) + 1);
```

Comment:
not a big issue but probably avoid modifying things not related to the PR

Reply:
Fig said it was alright.

Reply:
yea just fyi @Dragorn421 stuff like this is more than fine. some people forget to run the formatter all the time, so when it runs on the whole codebase it might catch things. and its good to include 
(sorry for the 20 day late response)